media/css/mozorg/webvision.scss

```diff
@@ -0,0 +1,122 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+$font-path: '/media/fonts';
+$image-path: '/media/protocol/img';
+
+@import '~@mozilla-protocol/core/protocol/css/includes/lib';
+@import '~@mozilla-protocol/core/protocol/css/components/sidebar-menu';
+@import '~@mozilla-protocol/core/protocol/css/templates/main-with-sidebar';
+
+.c-webvision-article {
+    h2 {
+        @include text-title-sm;
+    }
+
+    h3 {
+        @include text-title-xs;
+    }
+
+    h4 {
+        @include text-title-2xs;
+    }
+
+    h1,
+    h2,
+    h3,
+    h4 {
+        a {
+            position: relative;
+            transition: 2s;
+            visibility: hidden;
+
+            &::after {
+                content: '#';
+                color: $color-marketing-gray-30;
+                position: absolute;
+                left: 5px;
+                transition: all 0.2s;
+                visibility: visible;
+            }
+
+            &:hover {
+                &::after {
+                    color: $color-blue-50;
+                    visibility: visible;
+                }
+            }
+        }
+    }
+
+    ul {
+        li {
+            margin: $spacing-sm $spacing-lg;
+            list-style-type: disc;
+        }
+    }
+}
+
+.c-webvision-article-summary {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+}
+
+.c-webvision-article-full {
+    display: flex;
+    flex-direction: column;
+    position: relative;
+    justify-content: space-between;
+
+    @media #{$mq-md} {
+        flex-direction: row;
+
+        .c-webvision-article {
+            margin-left: 30px;
+        }
+    }
+
+    .mzp-c-sidemenu {
+        position: relative;
+    }
+
+    .webvision-sidebar {
+        position: sticky;
+        max-height: 100vh;
+        top: 0;
+        left: 0;
+        overflow-y: auto;
+        overflow-x: hidden;
+    }
+
+    .mzp-c-sidemenu-main {
+
+        @media #{$mq-md} {
+            padding: 100px 0;
+        }
+
+        @media #{$mq-xl} {
+            width: 350px;
+        }
+    }
+
+    .toc {
```

Comment:
This rule can be reduced. The second media query overrides the first so the first one is redundant. The second one then sets the same width as the base rule, so that's redundant as well. And then I'm not sure this element needs a width at all, it seems to look the same without it (I think it gets a width from Protocol?)

So I think all we really need here is some indentation on the nested lists. This could be reduced to something like:
```
.toc li ul {
    margin-left: $spacing-md;
}
```

Reply:
On second thought, if I add the style above the links go right to the edge of the scrollbar so maybe the whole thing needs a bit of padding, maybe:

```
.toc {
    padding: $spacing-md;
}
```

Reply:
I'll keep it as `padding: $spacing-xs` as I think that's good enough padding (from what I see) and it's not hitting any edges for me